magento/adobe-stock-integration
# Fix static tests failures on 2.0-develop per new rules

### Description (*)


### Fixed Issues (if relevant)

N/A

### Manual testing scenarios (*)
1.Verify MediaGallery messages displayed correctly,
2. Verify Adobe Stock Configuration displayed correctly
3. Verify Adobe stock sign in sign out functionality and styles.
